doc/source/programs/gdal_raster_pixelinfo.rst

```diff
@@ -0,0 +1,238 @@
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Return information on a pixel of a raster dataset
+
+.. Index:: gdal raster pixelinfo
+
+Synopsis
+--------
+
+.. program-output:: gdal raster pixelinfo --help-doc
+
+Description
+-----------
+
+:program:`gdal raster pixelinfo` provide a mechanism to query information about
+a pixel given its location in one of a variety of coordinate systems.
+
+It supports outputting either as GeoJSON or CSV.
+
+The following items will be reported (when known):
+
+- Input coordinates
+- Input coordinates converted to column, line
+- Pixel value per selected band(s), with unscaled value when offset/scale is associated to the band
+- For VRT files, which file(s) contribute to the pixel value.
+
+The following options are available:
+
+Standard options
+++++++++++++++++
+
+.. option:: -f, --of, --format, --output-format geojson|csv
+
+    Which output format to use. Default is GeoJSON.
+
+.. include:: gdal_options/oo.rst
+
+.. include:: gdal_options/if.rst
+
+.. option:: -b <band>
+
+    Selects a band to query. Multiple bands can be listed. By default all
+    bands are queried.
+
+.. option:: -p, --pos, --position <column,line> or <X,Y>
+
+    Required.
+    This can be specified either as an option, a positional value after the
+    dataset name, or when called from :program:`gdal`, as (space separated)
+    values provided on the standard input.
+
+    By default, when :option:`--position-type` is not specified, or set to ``pixel``,
+    this is a column, line tuple (possibly with fractional part).
+    If :option:`--position-type` is set to ``geoloc``, this is a georeferenced
+    coordinate expressed in the CRS of the dataset.
+    If :option:`--position-crs` is specified, this is a georeferenced
+    coordinate expressed in the CRS defined by :option:`--position-crs`.
+
+    X means always longitude or easting, Y means always latitude or northing.
+
+    Several x,y tuples may be specified.
+
+.. option:: --position-type pixel|geoloc|user-crs
+
+    Type of position:
+
+    - ``pixel`` means that the position is set as column, line (default)
+
+    - ``geoloc`` means that the position is a georeferenced
+      coordinate expressed in the CRS of the dataset.
+
+    - ``user-crs`` means that the position is a georeferenced
+      coordinate expressed in CRS defined by :option:`--position-crs`.
+
+.. option:: --position-crs <srs_def>
+
+    The coordinate system of the input x, y location.
+
+.. option:: -r, --resampling nearest|bilinear|cubic|cubicspline
+
+    Select a sampling algorithm. The default is ``nearest``.
+
+    The available methods are:
+
+    - ``nearest`` applies a nearest neighbour.
+
+    - ``bilinear`` applies a bilinear convolution kernel.
+
+    - ``cubic`` applies a cubic convolution kernel.
+
+    - ``cubicspline`` applies a B-Spline convolution kernel.
+
+.. option:: --ovr, --overview <index>
+
+    Query the (overview_level)th overview (overview_level=0 is the 1st overview),
```

Comment:
overview_level=1 seems more intuitive for the 1st overview, but I'm guessing this would create an inconsistency somewhere?

Reply:
- gdallocationinfo -ovr use 1-based index
- gdal raster footprint --overview use 0-based index (like the GDAL API)
- gdal_translate -ovr and gdalwarp -ovr use 0-based index